webapp/crashstats/crashstats/management/commands/verifyprocessed.py

```diff
@@ -76,35 +72,30 @@ def check_elasticsearch(supersearch, crash_ids):
 
 def check_crashids_for_date(firstchars_chunk, date):
     """Check crash ids for a given firstchars and date"""
-    s3_crash_source = build_instance_from_settings(socorro_settings.CRASH_SOURCE)
-    s3_crash_dest = build_instance_from_settings(socorro_settings.S3_STORAGE)
+    crash_source = build_instance_from_settings(socorro_settings.CRASH_SOURCE)
+    crash_dest = build_instance_from_settings(socorro_settings.STORAGE)
 
     supersearch = SuperSearchUnredacted()
 
     missing = []
 
     for firstchars in firstchars_chunk:
         # Grab all the crash ids at the given date directory
-        raw_crash_key_prefix = RAW_CRASH_PREFIX_TEMPLATE % (date, firstchars)
-
-        page_iterator = s3_crash_source.connection.list_objects_paginator(
-            bucket=s3_crash_source.bucket,
-            prefix=raw_crash_key_prefix,
+        page_iterator = crash_source.list_objects_paginator(
+            prefix=f"v1/raw_crash/{date}/{firstchars}",
         )
 
         for page in page_iterator:
             # NOTE(willkg): Keys here look like /v2/raw_crash/ENTROPY/DATE/CRASHID or
             # /v1/raw_crash/DATE/CRASHID
```

Comment:
This should get updated--there's no ENTROPY anymore. The only option is `/v1/raw_crash/DATE/CRASHID`. I missed this when I nixed the ENTROPY from the keys.